src/status_im/subs/general.cljs

```diff
@@ -151,3 +151,26 @@
  :<- [:toasts]
  (fn [toasts [_ toast-id & cursor]]
    (get-in toasts (into [:toasts toast-id] cursor))))
+
+(re-frame/reg-sub :currencies/categorized
+ :<- [:currencies]
```

Comment:
Shouldn't the currencies preferences live under `:profile`? 🤔
CC: @J-Son89 @smohamedjavid

Reply:
No, @ulisesmac. This is the list of currencies we show in the settings for user selection and is also used to get the currency symbol for a selected currency to display across the app.

Previously we used an outdated list stored locally in `utils.currency` ns. Now, we fetch from `status-go` (which is moved from `status-desktop`) to maintain the list same across the platforms.

Reply:
Thanks for the clarification!